docs/index.md

```diff
@@ -21,25 +21,49 @@ DDEV by default uses ports 80 and 443 on your system when projects are running.
 
 ## Installation
 
-### Docker Installation
+### Docker or Colima Prerequisite
 
 Docker or an alternative is required before anything will work with DDEV. This is pretty easy on most environments; see the [docker_installation](users/docker_installation.md) page to help sort out the details, especially on Windows and Linux. It is not required to install docker-compose because DDEV uses its own private version.
 
-### Homebrew - macOS/Linux
+### macOS (Homebrew)
 
-For macOS (both amd64 and arm64) and Linux users, we recommend installing and upgrading via [Homebrew](https://brew.sh/) (macOS) or [Homebrew on Linux](https://docs.brew.sh/Homebrew-on-Linux) (Linux): `brew install drud/ddev/ddev`.
+For macOS (both amd64 and arm64) users, we recommend installing and upgrading via [Homebrew](https://brew.sh/): `brew install drud/ddev/ddev`.
 
-As a one-time initialization, run `mkcert -install`. Linux users may have to take additional actions as discussed below in [Linux `mkcert -install` additional instructions](#Linux-mkcert--install-additional-instructions).
+As a one-time initialization, run `mkcert -install`.
 
 Later, to upgrade to a newer version of DDEV-Local, run `brew upgrade ddev`.
 
-To install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
+To install DDEV prereleases, subscribe to the "edge" channel with `brew install drud/ddev-edge/ddev` and to install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
 
-### Installation or Upgrade - Windows (WSL2)
+### Linux (Arch)
```

Comment:
It probably won't hurt to mention EndeavourOS and Manjaro because these distros are actually more popular now than Arch according to distrowatch.com.

Reply:
Maybe something along the lines of:
```
Linux (Arch-based systems such as EndeavourOS or Manjaro)

Reply:
You know you can use the "suggest" feature to just say what you suggest?

Reply:
Sorry, not following.  Are you saying I could `Start a review`?

Reply:
I think you are reviewing :) In any comment you can say what you think it ought to be. Then the other person can just click the button.

<img width="824" alt="Cursor_and__docs__Improve_installation_instructions__especially_for_WSL2_Linux_by_uberhacker_·_Pull_Request__3579_·_drud_ddev" src="https://user-images.githubusercontent.com/112444/154817135-09358c60-a6c8-4fbb-860c-d1f2724a3580.png">

Reply:
```suggestion
### Linux (Arch-based systems such as EndeavourOS or Manjaro)
```
Always learning new things. :)  It seems GitHub comes out with a new feature every other day.

Reply:
It's so much more efficient than making a comment then somebody having to make a commit.